docs_user/modules/proc_migrating-ceph-mds.adoc

```diff
@@ -174,15 +180,16 @@ placement:
   label: mds
 ----
 
-. Use the `ceph orchestrator` to apply the new MDS spec: it results in an
-increased number of mds daemons:
+. Use the `ceph orchestrator` to apply the new MDS spec:
 +
 ----
 $ sudo cephadm shell -m mds.yaml -- ceph orch apply -i /mnt/mds.yaml
 Scheduling new mds deployment …
 ----
++
+This results in an increased number of MDS daemons.
 
-. Check the new standby daemons temporarily added to the cephfs fs:
+. Check the new standby daemons that are temporarily added to the CephFS:
```

Comment:
Is it correct to just say “CephFS” here?

Reply:
technically "fs" stands for filesystem, but just leaving `CephFS` gives the idea: I think `fs` might result redundant, so I'm ok with your rewriting.